lib/constructs/common/lambda-functions.ts

```diff
@@ -302,10 +305,14 @@ export class TimetableFunctions extends cdk.Construct {
 export class SyllabusFunctions extends cdk.Construct {
 
     readonly getFunction: Function;
+    readonly updateFunction: Function;
+    readonly SyllabusBucket: s3.Bucket;
 
     constructor(scope: cdk.Construct, id: string, props: FunctionsProps) {
         super(scope, id);
-
+        const SyllabusBucket = new s3.Bucket(this,'waseda-syllabus',{
+            bucketName:'waseda-syllabus'
```

Comment:
Creating a bucket instance here will deploy a new bucket in the cloud. We already have a bucket for that (in syllabus data pipeline), you can pass it into this construct from the props.

Reply:
The bucket that needs to use("waseda-syllabus") doesn't exist in the syllabus pipeline, so I newed it in the SyllabusSyncPipeline as DataSource. But it seems to fail, so do I use "waseda-syllabus-prod" or do some other changes?